docs/reference/api/docker_remote_api_v1.21.md

```diff
@@ -51,6 +51,10 @@ List containers
                  "Created": 1367854155,
                  "Status": "Exit 0",
                  "Ports": [{"PrivatePort": 2222, "PublicPort": 3333, "Type": "tcp"}],
+                 "Labels": {
+                         "com.example.version": "15.04 Vivid Vervet",
+                         "com.example.date": "2015-04-23"
+                 },
```

Comment:
I think it would be good if we used the same example labels as were used in the other locations, so that we are consistent. This is what was added in the original PR; https://github.com/docker/docker/pull/9882/files#diff-c9bf94fbf101ffe50004bd58d69dde34